main.py

```diff
@@ -1,9 +1,18 @@
+"""Flask app."""
 from flask import Flask
-app = Flask(__name__)
 
-@app.route("/")
+APP = Flask(__name__)
+
+"""URL Routing list."""
```

Comment:
should this be at the top?

Reply:
> should this be at the top?

Dont think so, the routing list is below it?

Reply:
What is this docstring for? Typically they would be after a class or function definition or at the start of the file. This one doesn't seem to follow any of these.

Reply:
> What is this docstring for? Typically they would be after a class or function definition or at the start of the file. This one doesn't seem to follow any of these.

Pylint told me to add it in originally to get the file up to the standard. However I've just removed it and re-ran it and it seems fine.  I'll re-commit to reflect this!